### DISCUSSION TITLE
I see an extra bar depending on order of operations for (open applications, set layout to tabbed)

### DISCUSSION THREAD

**Discussion by kurtlieber** [i3/i3#5709] [Q&A]
This is probably an easy oversight on my part, but I don't know the right terms for what I'm seeing, so I don't know what to search for.

In a new workspace, if I open applications and then switch the layout to tabbed, everything works the way I want it to.

If, however, I set layout to tabbed and then open applications, I get an extra bar with the window class or title in it at the top of my screen, which is unwanted.

I use rofi for dmenu and polybar for a status bar, so any native i3 bars are not there by intent on my part.

I've also attached two screenshots showing what I"m seeing.  Any help is appreciated.

Bar: https://i.imgur.com/5orf04m.png

No Bar: https://i.imgur.com/hCxhgyi.png

**Comment by orestisfl** [comment]
See tracking issue: https://github.com/i3/i3/issues/3001

To remove this container, you just need to move the window to the left/right which will move it out of the surrounding container
